go.mod

```diff
@@ -17,6 +17,7 @@ require (
 	github.com/karlseguin/ccache/v2 v2.0.8
 	github.com/kelseyhightower/envconfig v1.4.0
 	github.com/oklog/ulid/v2 v2.0.2
+	github.com/pressly/goose v2.7.0+incompatible
```

Comment:
Look at the README for the repo. It defaults to 2.7.x but we ideally want to be in 3.0 which is the actively maintained one. It will no longer have `vendored` dependencies which ultimately might make FOSSA happy :)

Reply:
Ah, I see. Thanks! Let me try.

Reply:
We are going to have to write our own migration library aren't we?